.circleci/config.yml

```diff
@@ -264,6 +266,12 @@ commands:
           key: *npm_cache_key
           paths:
             - ./node_modules
+  run-tests:
+    description: "Run all tests"
+    steps:
+      - run:
+          name: ng test
+          command: ng test ddp-sdk --watch=false --browsers ChromeHeadlessNoSandbox
 
```

Comment:
ng tests behaves weird when you run tests on everything. Keeps relaunching browser. Might be correctable, but alll tests reside at least for now under ddp-sdk